src/main/java/io/passport/server/controller/ModelController.java

```diff
@@ -49,6 +49,7 @@ public ResponseEntity<List<Model>> getAllModels(@RequestParam(value = "studyId",
         return ResponseEntity.ok(models);
     }
 
+
```

Comment:
Newline left from deleted content. You can remove them along with the other marked newlines.